apisix/plugins/ext-plugin/init.lua

```diff
@@ -935,13 +935,14 @@ end
 
 local runner
 local function setup_runner(cmd)
-    runner = spawn_proc(cmd)
 
     ngx_timer_at(0, function(premature)
         if premature then
             return
         end
 
+        runner = spawn_proc(cmd)
+
```

Comment:
The `spawn_proc` will call some apis from `lua-resty-core`

> The communication APIs do not work in phases which do not support yielding, such as init_worker_by_lua* or log_by_lua*, because there is no way to yield the current light thread to avoid blocking the OS thread when communicating with processes in those phases.

ref: https://github.com/openresty/lua-resty-core/blob/v0.1.28/lib/ngx/pipe.md#description

Reply:
why it used to work without any problem in the previous version?

Reply:
I don't know the reason now. But doing this can fix the CI.

Reply:
I think you should find the reason?

Reply:
[this test-case](https://github.com/apache/apisix/blob/f79c4eba51a96ef7223a045a0d39465c47ab4b71/t/plugin/ext-plugin/sanity2.t#L57) will failed because the spawned processs already has exited without output data in https://github.com/apache/apisix/blob/f79c4eba51a96ef7223a045a0d39465c47ab4b71/apisix/plugins/ext-plugin/init.lua#L949